### PR TITLE
Using Semantic Versioning for Chart version.

### PR DESCRIPTION
**Pull request by kinderyj**
I got an exception when trying to push this charts to the my server. 
It show me that the chart version shoud use the Semantic Versioning. If the version is not "Semantic Versioning"， the chart server will not recognize it.
Semantic version Guide: https://semver.org/

### CODE REVIEW

**Comment by functicons** [comment]
@kinderyj Could you sign the CLA?

**Comment by kinderyj** [comment]
> @kinderyj Could you sign the CLA?

sure, I will sign the CLA.

**Comment by kinderyj** [comment]
> https://cla.developers.google.com/

@googlebot I signed it!

**Comment by functicons** [comment]
Seems it didn't work, are you using a different email account in your git commit than your github account?

**Comment by kinderyj** [comment]
> Seems it didn't work, are you using a different email account in your git commit than your github account?

Sorry, Let me try again, I will re-create a pr with a new one.
